src/main/java/at/hannibal2/skyhanni/data/MaxwellAPI.kt

```diff
@@ -52,10 +52,14 @@ object MaxwellAPI {
     private var powers = mutableListOf<String>()
 
     private val patternGroup = RepoPattern.group("data.maxwell")
-    private val chatPowerpattern by patternGroup.pattern(
+    private val chatPowerPattern by patternGroup.pattern(
         "chat.power",
         "§eYou selected the §a(?<power>.*) §e(power )?for your §aAccessory Bag§e!"
     )
+    private val chatPowerUnlockedPattern by patternGroup.pattern(
+        "chat.power.unlocked",
+        "§eYou permanently unlocked the §a(?<power>.*) §epower!"
```

Comment:
I would prefer to use the other message, just in case Hypixel ever changes the current behavior of automatically switching to the newly unlocked power.
```suggestion
        "§eYour selected power was set to (?:§r)*§a(?<power>.*)(?:§r)*§e!"
```